new/collapse-mixlib-install-in-tk.md

```diff
@@ -0,0 +1,171 @@
+Type: Standards Track
+---
+
+# Collapse Mixlib Install Code Paths in Test Kitchen
+Test Kitchen currently supports two code paths for configuring, downloading and installing Chef and ChefDK on test instances. Both code paths use Mixlib Install but are implemented differently. In the past year, significant effort has been put into Mixlib Install to make it the canonical source for interacting with Chef Software Inc's release infrastructure and generating bootstrap installation scripts (install.sh and install.ps1).
+
+Mixlib Install's new API is currently used for downloads.chef.io and omnitruck.chef.io. It's also used for the chef-ingredient cookbook. Chef transparently implemented the new API and selected config options in Kitchen for testing purposes. It has proven valuable and easy to use. The feature set is limited at this time.
+
+## Motivation
+```
+As a Kitchen User,
+I want to use simple, clear configuration options,
+so that I can setup Kitchen config files with less complexity and obfuscation.
+```
+
+```
+As a Kitchen Developer,
+I want to maintain a single Mixlib Install code path,
+so that I can benefit from the latest API changes and reduce maintenance risk.
+```
+
+The current Mixlib Install code path used in Kitchen was developed specifically for Kitchen and is tightly coupled with Kitchen's provisioner options. It's referred to as the `ScriptGenerator`. The new API was developed to be used by any project that needs to query Chef product meta-data or generate an installation script. The new API takes into account the specifics of interacting with Chef Software Inc's release infrastructure (packages.chef.io).
+
+## Specification
+This work is separated into 3 phases.
+
+##### Phase 1 - Incrementally add new transparent configuration options
+* All existing configuration options behave as expected
+* Build feature parity with the new options to match existing functionality
+* Setting `product_name` will trigger the new options use cases
+* Maintain both Mixlib Install code paths
+* Document new options in the Kitchen repo
+
+##### Phase 2 - Provide option deprecation warnings and "how-tos"
+* All existing configuration options still behave as expected
+* User is clearly warned when using options that will be deprecated
+* Deprecation warnings include explanations and resolution instructions
+* Inform the community of the upcoming Kitchen changes and reasoning
+
+##### Phase 3 - Make the switch
+* Raise errors along with instructions when using deprecated options 
+* `ScriptGenerator` code path removed from Kitchen
+* docs.chef.io and kitchen.ci documentation updates
+* Inform the community when Kitchen is released
+
+### Phase 1 Details
+Below is a proposal of how the new options `product_version` and `install_strategy` replace `require_chef_omnibus`. It outlines requirements to maintain feature parity with Kitchen's current configuration options. Since `require_chef_omnibus` is a complex option it warranted this level of detail.
```

Comment:
Use the active voice for the sentence starting "Below is a proposal.." e.g. "The following describes our proposal to replace `require_chef_omnibus` with the new options..."